templates/common/cloud-platform-alt-dns/files/coredns.yaml

```diff
@@ -24,6 +25,12 @@ contents:
       - name: conf-dir
         hostPath:
           path: "/etc/coredns"
+      - name: nm-resolv
+        hostPath:
+          path: "/var/run/NetworkManager"
```

Comment:
Does it work as expected if Pod starts before this exists? Do we need to pass the whole directory or can we just pass the file? Name says `nm-resolv` so I would expect the path to point just to resolv.conf (unless this is a hack in case Pod starts before resolv.conf file exists, but after the directory has been created)

Reply:
I don't think we have encountered the situation in our testing yet where the pod is unable to find `resolv.conf` at this location. It seemed like a good idea to not assume that the file is always going to be present when the pod is started.